e2e-tests/content-strings/error-messages.js

```diff
@@ -141,6 +141,15 @@ export const ERROR_MESSAGES = {
           },
         },
       },
+      ABOUT_GOODS_OR_SERVICES: {
+        [FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES.DESCRIPTION]: {
+          IS_EMPTY: 'Enter the goods or services you will supply to the buyer',
+          ABOVE_MAXIMUM: 'The goods or services you will supply to the buyer cannot be more than 1000 characters.',
```

Comment:
is there a reason this one has a full stop and the others don't? should we keep it standard for all with/without punctuation at the end unless its a question

Reply:
Great spot